### PR TITLE
is1594 fix and re-activate e2e testing

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

- [x] e2e runs in host ( partially implements #1367)
    - installs/uninstall insecure registry (see [e2e.bash](ci/github/system-testing/e2e.bash) )
- [x] e2e flow is reviewed and updated
- [x] e2e is **reactivated** in github actions and **removed** from travis
- Fixes bugs in [postgres migration scripts](packages/postgres-database/src/simcore_postgres_database/cli.py) related to unhandled exceptions

image

## Related issue number

solves #1594 
part of #1367

## How to test

```cmd

make devenv clean

/bin/bash ./ci/github/system-testing/e2e.bash install
/bin/bash ./ci/github/system-testing/e2e.bash test
/bin/bash ./ci/github/system-testing/e2e.bash clean_up

make clean
```

## Checklist

- [x] Did you change any service's API? Then make sure to bundle document and upgrade version (``make openapi-specs``, ``git commit ...`` and then ``make version-*``)
- [x] Unit tests for the changes exist
- [x] Runs in the swarm
- [x] Documentation reflects the changes
- [x] New module? Add your github username to [.github/CODEOWNERS](.github/CODEOWNERS)

### CODE REVIEW

File: .github/workflows/ci-testing-deploy.yml [hunk 1]
```diff
@@ -917,7 +923,7 @@ jobs:
         integration-test-sidecar,
         integration-test-simcore-sdk,
         system-test-swarm-deploy,
-        #system-test-e2e,
+        system-test-e2e,
```
Comment (GitHK) [inline .github/workflows/ci-testing-deploy.yml:926]: 👍

File: tests/e2e/tutorials/sleepers.js [hunk 2]
```diff
@@ -18,7 +19,7 @@ const {
 const templateName = "Sleepers";
 
 async function runTutorial() {
-  const tutorial = new tutorialBase.TutorialBase(url, user, pass, newUser, templateName);
+  const tutorial = new tutorialBase.TutorialBase(url, user, pass, newUser, templateName, enableDemoMode);
```
Comment (odeimaiz) [inline tests/e2e/tutorials/sleepers.js:22]: Please, move the args processing step to the TutorialBase or add the same thing in the Mattward tutorial

File: tests/e2e/utils/auto.js [hunk 3]
```diff
@@ -190,12 +197,12 @@ async function runStudy(page, waitFor = 0) {
   try {
     await responsesQueue.waitUntilResponse("/start");
   }
-  catch(err) {
+  catch (err) {
```
Comment (odeimaiz) [inline tests/e2e/utils/auto.js:200]: Not sure if all these whitespaces were necessary

Comment (pcrespov) [reply tests/e2e/utils/auto.js:200]: That was the autoformatter ...  :-) I can undo that

File: tests/e2e/tutorials/sleepers.js (hunk 2 above)
Comment (pcrespov) [reply tests/e2e/tutorials/sleepers.js:12]: good point.

File: tests/e2e/utils/auto.js (hunk 3 above)
Comment (ignapas) [reply tests/e2e/utils/auto.js:198]: The autoformatter is useless xD It's destroying the ``for`` loop classic look...

**Pull request merged**